libs/fujitsu-thermal-printer/vitest.config.ts

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from '../../vitest.config.shared.mjs';
+
+export default defineConfig({
+  test: {
+    coverage: {
+      thresholds: {
+        lines: 0,
+        branches: 0,
```

Comment:
Intentionally 0?

Reply:
Yeah, [it was 0 before](https://github.com/votingworks/vxsuite/pull/5806/files#diff-0c4f135684fe237c2c612fab053296b19510cf0344c661adc64f443a35c6bfdf).